### PR TITLE
Add script to comment on all open PRs

### PR DESCRIPTION
**Pull request by t-eckert**
Changes proposed in this PR:
- Adds a script for commenting on all open PRs to a repository

How I've tested this PR:

I ran the script against my own repository and it commented on all the open pull requests successfully. You can see the results here: https://github.com/hashicorp/issue-shepherd/pull/1

How I expect reviewers to test this PR:

You can also run the script against a sample repository.

### CODE REVIEW

File: charts/consul/hack/mass-pr-commenter/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Mass PR Commenter Script
+
+This script allows the user to comment on every open pull request in a given repository.
+
+## Prerequisites
```
Comment (thisisnotashwin) [inline charts/consul/hack/mass-pr-commenter/README.md:5]: :driveby-comment: I think the `gh` CLI is another pre-requisite?

Comment (t-eckert) [reply charts/consul/hack/mass-pr-commenter/README.md:5]: Yes! It is.

**Comment by t-eckert** [comment]
I am going to close this PR because I don't see a need to keep it in this repository. Thank you for getting some eyes on it. I will run it after we do the `master` to `main` change.
